test_mocks.py

```diff
@@ -0,0 +1,8 @@
+import test_fixtures as fixtures
+
+def fetch_tags_summary() -> dict:
+    return fixtures.test_fetch_machines_grouped_by_tags_tags
+
```

Comment:
can you check the coding standards pass here ? I believe outside a class you should have two new lines between functions. try to run `flake8` or check in the ci